src/pages/debrief/cat-cpc/debrief.cat-cpc.page.ts

```diff
@@ -135,6 +135,10 @@ export class DebriefCatCPCPage extends BasePageComponent {
     }
   }
 
+  isTerminated(): boolean {
+    return this.outcome !== TestOutcome.PASS && this.outcome !== TestOutcome.FAIL;
```

Comment:
are we not able to use TestOutcome.Terminated found at 'src/modules/tests/tests.constants.ts'?